de-lib/src/main/java/org/iplantc/de/diskResource/client/gin/factory/DiskResourceSelectorFieldFactory.java

```diff
@@ -22,7 +22,7 @@
 
     FileFolderSelectorField createFilteredFileFolderSelector(List<InfoType> infotypes);
 
-    MultiFileSelectorField creaeteMultiFileSelector(boolean allowFolderSelect);
+    MultiFileSelectorField creaeteMultiFileSelector(boolean allowFolderSelect, String gridEmptyText);
```

Comment:
Also minor - create is spelled wrong with an extra "e" in `creaeteMultiFileSelector`